docs/streaming/sources/azure-event-hub.mdx

```diff
@@ -1,12 +1,91 @@
 ---
 title: "Azure Event Hub"
+description: "This guide provides step-by-step instructions to set up and use Azure Event Hub with Mage. By following these instructions, 
+you can create a new streaming pipeline in Mage, authenticate with Azure Event Hub, and configure your data loader, transformer, 
+and exporter blocks to process and manage your data efficiently."
 ---
 
-## Config
+**Start Mage**
 
-```yaml
+1. Open Mage in your browser and then navigate to the pipelines page.
+2. Create a new pipeline by clicking the “+ New” button and then select “Streaming.”
+3. Enter a name for the pipeline and click “Create.”
+
+**Authentication**
+To authenticate with Azure Event Hub, complete the following actions:
+
+1. Configure your io_config.yml file with the account variables below. If you need further information about how to store secrets in 
+Mage check out the [documentation](https://docs.mage.ai/production/deploying-to-cloud/secrets/Azure).
+
+```YAML
+  AZURE_CLIENT_ID: "{{ env_var('AZURE_CLIENT_ID') }}"
+  AZURE_CLIENT_SECRET: "{{ env_var('AZURE_CLIENT_SECRET') }}"
+  AZURE_STORAGE_ACCOUNT_NAME: "{{ env_var('AZURE_STORAGE_ACCOUNT_NAME') }}"
+  AZURE_TENANT_ID: "{{ env_var('AZURE_TENANT_ID') }}"
+```
+1. Add a data loader block to your pipeline and then select Azure Event Hub as the source.
```

Comment:
nit: quotes around "Azure Event Hub"

Reply:
Made this change see picture 
<img width="756" alt="Screenshot 2024-07-09 at 10 57 50 AM" src="https://github.com/mage-ai/mage-ai/assets/92221031/05fd8d27-8ef2-4fb0-8229-c39c868998bd">